new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/IntegrationTestWatcher.java

```diff
@@ -313,6 +314,7 @@ public void afterAll(ExtensionContext context) {
     } else {
       logger.info("Starting cleanup after test class");
       CleanupUtil.cleanup(namespaces);
+      IstioUtils.uninstallIstio();
```

Comment:
This should be moved inside cleanuputil.java

Reply:
This is not a general cleanup task, so I put hem into  IstioUtils.

Reply:
@anpanigr we need to keep all cleanup code in one place.
You can put this this as a first step in CleanupUtil.java

  public static void cleanup(List<String> namespaces) {
    try {
      IstioUtils.uninstallIstio();
      // If namespace list is empty or null return
      if (namespaces == null || namespaces.isEmpty()) {
        logger.info("Nothing to cleanup");
        return;
      }